developer_guide/migration/index.rst

```diff
@@ -45,6 +45,16 @@ If the patch is a javascript fix, please clear your browser cache before testing
     We set any other dependencies to their exact patch versions to avoid compatibility issues.
     We strongly advise you to add the composer.lock in your versioning system.
 
+
+.. note::
+   
+   If you getting a 500 error after an upgrade and clear cache aint working try to clear apc cache and restart the server.
```

Comment:
The 500 error is very likely to occur since a major migration involves changes in the database schema. As this schema is stored in APC, this cache **has** to be cleared. It has to be done via a PHP page stored in your web directory, so you can access it via the browser. You should not meet any problem via the CLI since APC cache for CLI is only enabled if `apc.enable_cli = 1` and is isolated from Apache's one (two very distinct processes).

So we end up with two options:
- restart Apache
- or, clear APC cache calling an `apc.php` containing:
```
<?php
apc_clear_cache();
apc_clear_cache('user');
apc_clear_cache('opcode');
```